common/djangoapps/student/views.py

```diff
@@ -874,20 +849,6 @@ def change_enrollment(request, check_access=True):
 
         # Otherwise, there is only one mode available (the default)
         return HttpResponse()
-
-    elif action == "add_to_cart":
```

Comment:
@bradenmacdonald I'm not familiar with the reproduction steps for this issue, but @chrisndodge should have more details.